src/views/Onboarding/components/GooglePlacesAutocomplete.jsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { compose, withProps } from "recompose";
+import { withScriptjs } from "react-google-maps";
+import PlacesAutocomplete from "react-places-autocomplete";
+import TextField from "@material-ui/core/TextField/TextField";
+import InputAdornment from "@material-ui/core/InputAdornment/InputAdornment";
+import Search from "@material-ui/icons/Search";
+import withStyles from "@material-ui/core/styles/withStyles";
+
+
+const styles = theme => ({
+    textInput: {
+        width: 400,
+    },
+    cssLabel: {
+        transform: 'translate(12px, 14px) scale(1)',
+        '&$cssFocused': {
+            transform: 'translate(14px, -6px) scale(0.75)',
+        },
+    },
+    cssFocused: {},
+    suggestionContainer: {
+        margin: '5px 0',
+        borderRadius: 10,
+        overflow: 'hidden',
+        boxShadow: 'none',
+        border: '1px solid #CCC',
+        position: 'absolute',
+        zIndex: 999,
+        background: '#FFF',
+    },
+    suggestionActive: {
+        backgroundColor: '#258ecc',
+        color: '#FFF !important',
+    },
+    suggestion: {
+        color: '#626262',
+        padding: '12px 5px',
+        cursor: 'pointer',
+        borderBottom: '1px solid #CCC',
+    },
+});
+
+const PlacesAutocompleteImpl = compose(
+  withProps({
+      loadingElement: <div style={{ height: `100%` }}>Loading...</div>,
+      containerElement: <div style={{ height: `400px` }} />,
+  }),
+  window.google ? null : withScriptjs
+)(props =>
+  <PlacesAutocomplete
+    value={props.addressInput}
+    onChange={props.handleChange}
+    onSelect={props.handleSelect}
+    className={props.classes.textInput}
+    searchOptions={{
+        componentRestrictions: {country: "nl"},
+    }}
+  >
+      {({ getInputProps, suggestions, getSuggestionItemProps }) => (
+        <div>
+            <TextField
+              type="text"
+              variant="outlined"
+              name="search"
+              className={props.classes.textInput}
+              label="Zoek in je plaats"
```

Comment:
label should be "Zoek naar speeltuinen bij je in de buurt"

Reply:
Done